constants.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+DB_NAME = os.getenv("db_name", "postgres")
+DB_PASSWORD = os.getenv("db_pwd", "password")
+DB_USER = os.getenv("db_user", "postgres")
+HOST = os.getenv("host", "localhost")
+PORT = os.getenv("port", "5432")
```

Comment:
```suggestion
HOST = os.getenv("db_host", "localhost")
PORT = os.getenv("db_port", "5432")
```

For consistency with the other three, and to avoid name collisions.